touchforms/formplayer/static/formplayer/script/test_vendor/underscore.js

```diff
@@ -0,0 +1,1548 @@
+//     Underscore.js 1.8.3
```

Comment:
would love to switch to lo-dash. yea you're right it's only for testing, i should change it to 1.4.4 since that's what's loaded on the page, hence i can't actually use `findIndex`